pymc3/sampling.py

```diff
@@ -24,6 +27,8 @@
 import sys
 sys.setrecursionlimit(10000)
 
+assert Any # ugly way to make pyflakes ignore "unused" Any import
```

Comment:
Better to just not import Any. It's only used in the commented variable annotation